### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by whitewhidow**
there is no .phar extension

### CODE REVIEW

**Comment by xabbuh** [comment]
This change looks wrong to me (see https://github.com/sensiolabs-de/deprecation-detector/blob/master/box.json#L2). Did you actually run `box build` before?

**Comment by whitewhidow** [comment]
i did not run `box build` as it was optional,   i just cloned it and ran it

File: README.md [hunk 1]
```diff
@@ -46,8 +46,8 @@ $ box build
 If you want to call the deprecation-detector globally, it needs to be placed in your `PATH`. For example with:
 
 ```bash
-$ sudo chmod a+x deprecation-detector.phar
-$ sudo mv deprecation-detector.phar /usr/local/bin/deprecation-detector
+$ sudo chmod a+x deprecation-detector
```
Comment (theofidry) [inline README.md]: if you did not run the `box build` as it is optional, `deprecation-detector` is under the `bin` directory

File: README.md [hunk 2]
```diff
@@ -46,8 +46,8 @@ $ box build
...
+$ sudo mv deprecation-detector /usr/local/bin/deprecation-detector
```
Comment (theofidry) [inline README.md]: is the sudo required here actually?

**Pull request closed without merging**